zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -196,7 +205,15 @@ public static List<Host> getTheTrainHosts() {
     }
 
     public static List<Host> getWebsiteHosts() {
-        return Arrays.asList(StringUtils.split(defaultIfBlank(getValue("website_url"), DEFAULT_WEBSITE_URL), ","))
+        String defaultLowercaseUrl = "website_url";
```

Comment:
We have discussed this together and renamed this productionUrl. It will look at the default first, but if it finds this url set to a value it will use the production URLs

Reply:
I still don't understand what this is supposed to do, I have a few questions:

1. How come this variable needs to change to `legacy_cache_api_url` when the API is enabled?
2. How are `legacy_cache_api_url` or `website_url` being populated?
3. Why is the new variable called `productionUrl`? Even without knowing what it does, it feels wrong to name it after an environment, the code should be environment-agnostic.

Maybe this is just me not understanding what this function is supposed to do, or how it works.

Reply:
1. the DEFAULT_LEGACY_CACHE_URL is for locally and "legacy_cache_api_url" is for running in sandbox, staging and production. we need to set a secret here.

Reply:
2. we read them from a secret. I don't know yet how we set a secret for sandbox or staging or production.